cleantalk.php

```diff
@@ -141,6 +141,14 @@
     \Cleantalk\Antispam\DisableComments::getInstance();
 }
 
+// Email encoder
+if (
+    $apbct->key_is_ok &&
+    ( ! is_admin() || apbct_is_ajax() ) &&
+    $apbct->settings['data__email_decoder'] ) {
+    \Cleantalk\Antispam\EmailEncoder::getInstance();
```

Comment:
why such a construction, the object is not used further...

Reply:
The instance is used into the class code by $this link